### PR TITLE
Add tests for "browsingContext.domContentLoaded" event.

### PR DESCRIPTION
**Pull request by lutien**
Add tests for [browsingContext.domContentLoaded](https://w3c.github.io/webdriver-bidi/#event-browsingContext-domContentLoaded) event.

### CODE REVIEW

**Comment by lutien** [comment]
Hey @sadym-chromium, when you have a minute, could you have a look at this PR? 
It was already reviewed from our side.

File: webdriver/tests/bidi/browsing_context/dom_content_loaded/dom_content_loaded.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+    await bidi_session.browsing_context.navigate(
+        context=new_tab["context"], url=test_page_same_origin_frame
+    )
+
+    wait = AsyncPoll(
+        bidi_session, message="Didn't receive dom content loaded events for frames"
+    )
+    await wait.until(lambda _: len(events) >= 2)
+    assert len(events) == 2
```
Comment (sadym-chromium) [inline webdriver/tests/bidi/browsing_context/dom_content_loaded/dom_content_loaded.py:76]: In our implementation, the created iFrame first is created with `about:blank`, and then is navigated to the desired URL, meaning at this point there will be 3 events: 
1. parent with `test_page_same_origin_frame`.
2. frame with `about:blank`.
3. frame with `test_page`.

Comment (lutien) [reply webdriver/tests/bidi/browsing_context/dom_content_loaded/dom_content_loaded.py]: Alright, I've updated the test to make it not sensitive to these differences

File: webdriver/tests/bidi/browsing_context/dom_content_loaded/dom_content_loaded.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    await bidi_session.session.subscribe(events=[DOM_CONTENT_LOADED_EVENT])
+
+    await bidi_session.browsing_context.navigate(
+        context=new_tab["context"], url=test_page_same_origin_frame
+    )
+
+    wait = AsyncPoll(
+        bidi_session, message="Didn't receive dom content loaded events for frames"
+    )
+    await wait.until(lambda _: len(events) >= 2)
```
Comment (juliandescottes) [inline webdriver/tests/bidi/browsing_context/dom_content_loaded/dom_content_loaded.py:75]: Should we also update this? This relies on the current FF implementation where we know that the 2nd event is going to be the iframe one. Should we just wait until we get an event with url set to `test_page` ?

As a follow up maybe we need to update our other browsing_context event tests to stop relying on event counts. The pattern you are using here seems less flaky.

Comment (whimboo) [reply webdriver/tests/bidi/browsing_context/dom_content_loaded/dom_content_loaded.py:75]: Another solution is to filter out those `DOMContentLoaded` events that are coming from the initial `about:blank` for this particular test. We know that we can safely ignore that one.

Comment (lutien) [reply webdriver/tests/bidi/browsing_context/dom_content_loaded/dom_content_loaded.py:77]: Added filtering

**Review by sadym-chromium — APPROVED** [review]

File: webdriver/tests/bidi/browsing_context/dom_content_loaded/dom_content_loaded.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        events.append(data)
+
+    remove_listener = bidi_session.add_event_listener(
+        DOM_CONTENT_LOADED_EVENT, on_event
+    )
+
+    url = inline("<div>foo</div>")
+
+    # When navigation reaches complete state,
+    # we should have received already browsingContext.domContentLoaded event
```
Comment (whimboo) [inline webdriver/tests/bidi/browsing_context/dom_content_loaded/dom_content_loaded.py:28]: Just a nit: `we should have received a browsingContext.domContentLoaded event`

**Comment by lutien** [comment]
@sadym-chromium, thanks a lot for the quick review!
Going to close this PR now, it will be synced from Mozilla repo.

**Comment by lutien** [comment]
The upstream PR is https://github.com/web-platform-tests/wpt/pull/36821

**Pull request closed without merging**